docs/book/src/user/quick-start.md

```diff
@@ -150,11 +150,11 @@ a target [management cluster] on the selected [infrastructure provider].
 ### Install clusterctl
 The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
 
-{{#tabs name:"install-clusterctl" tabs:"linux,macOS,homebrew,Windows"}}
+{{#tabs name:"install-clusterctl" tabs:"Linux,macOS,homebrew,Windows"}}
 {{#tab linux}}
```

Comment:
```suggestion
{{#tab Linux}}
```
Let's also update those (a few more below). Seems to work anyway, but for consistency